crates/ruff_linter/src/checkers/ast/analyze/statement.rs

```diff
@@ -97,6 +97,9 @@ pub(crate) fn statement(stmt: &Stmt, checker: &mut Checker) {
             if checker.enabled(Rule::InvalidBoolReturnType) {
                 pylint::rules::invalid_bool_return(checker, name, body);
             }
+            if checker.enabled(Rule::InvalidLengthReturnType) {
+                pylint::rules::invalid_length_return(checker, name, body);
```

Comment:
Nit: You can pass the entire function definition instead of just passing the name and body
```suggestion
                pylint::rules::invalid_length_return(checker, function_def);
```